src/web/app/components/question-responses/gqr-rqg-view-responses/gqr-rqg-view-responses.component.html

```diff
@@ -32,8 +32,11 @@
   <div class="card top-padded">
     <div class="card-header bg-primary text-white" (click)="userExpanded[userInfo] = !userExpanded[userInfo]">
       {{ isGqr ? 'From' : 'To' }}: {{ userInfo }}
+      <span *ngIf="userToEmail[userInfo]">
+        [{{ userToEmail[userInfo] }}]
```

Comment:
Let's follow V6 where we should use `mailto` here.

https://www.w3schools.com/tags/tag_address.asp

Reply:
Alright will use that!